store/src/java-test/com/zimbra/cs/account/MockProvisioning.java

```diff
@@ -56,6 +57,9 @@
  * @author ysasaki
  */
 public final class MockProvisioning extends Provisioning {
+    static {
+      LC.zimbra_attrs_directory.setDefault(System.getProperty("user.dir") + "/conf/attrs");
```

Comment:
@rohan-ambasta I'm wondering if we even need this here at all.  Isn't the common pattern to call `MailboxTestUtil.initProvisioning()` first?  If so, that is already initializing `LC.zimbra_attrs_directory` correctly (using the method you described).

Should I just rebase out this change completely?